### ISSUE TITLE
Secret Registration page breaking

### ISSUE THREAD

**Issue by kevincolten** [RocketChat/Rocket.Chat#6755]
Rocket.Chat Version: 0.55.1
Running Instances: 1
DB Replicaset OpLog: 
Node Version: v4.8.1

The secret registration page is breaking since, at least, the latest update. You are left at the loading screen with this error in the console

```
Exception in template helper: TypeError: Cannot read property 'depend' of undefined
```

**Comment by mrgnr** [comment]
I just upgraded from 0.47.1 to 0.55.1 and am seeing a similar issue. The secret registration page just hangs with the loading animation. I don't see an exception, but I see the following in the logs:

```
Meteor ➔ publish null -> userId: null , arguments:  {}
Meteor ➔ method autoTranslate.getSupportedLanguages -> userId: null , arguments:  { '0': 'en' }
Meteor ➔ method listCustomSounds -> userId: null , arguments:  {}
Meteor ➔ method listEmojiCustom -> userId: null , arguments:  {}
Meteor ➔ method checkRegistrationSecretURL -> userId: null , arguments:  { '0': 'mysecrettoken' }
Meteor ➔ method UserPresence:online -> userId: null , arguments:  {}
Meteor ➔ publish roles -> userId: null , arguments:  {}
Meteor ➔ publish meteor_autoupdate_clientVersions -> userId: null , arguments:  {}
Meteor ➔ publish stream-notify-all -> userId: null , arguments:  { '0': 'updateCustomSound', '1': false }
Meteor ➔ publish stream-notify-all -> userId: null , arguments:  { '0': 'deleteCustomSound', '1': false }
Meteor ➔ publish activeUsers -> userId: null , arguments:  {}
Meteor ➔ publish userData -> userId: null , arguments:  {}
Meteor ➔ publish stream-notify-all -> userId: null , arguments:  { '0': 'public-settings-changed', '1': false }
```

I'm running on Ubuntu 16.04 with Node 4.5.0.

**Comment by MartinSchoeler** [comment]
@ggazzo is this related to the conversion from coffee of this file?
 https://github.com/RocketChat/Rocket.Chat/blob/e5ca5b88fa02f4f62db4e71fa8d333ba3b1df1ab/packages/rocketchat-ui/client/views/app/secretURL.js

**Comment by kimgb** [comment]
I am also experiencing this if you need any extra data. RocketChat 0.55.1, Node 4.5, Ubuntu 16.04.

**Comment by JamieH** [comment]
i'm having this too.

**Comment by jbguerraz** [comment]
without much surprise, same here with version 0.56.0

**Comment by MartinSchoeler** [comment]
Fixed by #7321
